py-polars/polars/cfg.py

```diff
@@ -64,6 +64,14 @@ def set_tbl_cols(cls, n: int) -> "Type[Config]":
         ----------
         n
             number of columns to print
+        
+        Examples
+        --------
+
+        >>> import polars as pl
+        >>> pl.cfg.Config.set_tbl_cols(1000)
+        >>> df = pl.DataFrame({str(i):[i] for i in range(100)})
+        >>> df
```

Comment:
In other examples we always show the result. But given that there are 100 columns it is too much to show, so maybe take a smaller number of columns and show the resulting table.